### DISCUSSION TITLE
Infinite scroll list

### DISCUSSION THREAD

**Discussion by dro-sh** [refinedev/refine#3260] [Q&A] [answered]
There is way to implement infinite scroll?

I has a list of people and I want to infinite scroll them on sidebar. Also I want use something like https://tanstack.com/virtual/v3

Only one idea is to save list on component state and then push more items then user on bottom of list, but I wouldn't like to manage local state

**Comment by alicanerdurmaz** [comment, marked as answer]
Hi @dro-sh, We have this feature in our backlog. You can follow progress on this [issue](https://github.com/refinedev/refine/issues/2854).

until this feature comes, if managing the state won't be a problem, you can follow this [example](https://refine.dev/docs/api-reference/antd/hooks/field/useSelect/#infinite-loading-example)
